src/indexed-reader.js

```diff
@@ -68,7 +68,7 @@ export class CarIndexedReader {
   /**
    * See {@link CarReader#getRoots}
    *
-   * @method
+   * @function
```

Comment:
what's with these changes? not a big deal I think but they _are_ methods; I don't think jsdoc4readme is going to hiccup on these but are they necessary changes?

Reply:
That's the linter that's done that, otherwise you get:

```
Invalid JSDoc tag (preference). Replace "method" JSDoc tag with "function". [Error/jsdoc/check-tag-names]
```

Apparently there's [no such tag](https://jsdoc.app/#block-tags) as `@method`, but `@function` is marked:

```
[@function](https://jsdoc.app/tags-function.html) (synonyms: @func, @method)
Describe a function or method.
```

Reply:
amusing, since it's right there in the `synonyms`

that's fine, if jsdoc4readme has a problem with it I should be able to address it there